boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_shared.dts

```diff
@@ -0,0 +1,132 @@
+/ {
+leds:	leds {
+		compatible = "gpio-leds";
+		rgb1_red: led_0 {
+			gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
+			label = "0 LED_RGB_RED CORE_APP";
```

Comment:
this has not been fixed. -1 on encoding "structured information" on label, this is not the purpose of such property. If needed downstream, add an overlay there until addressed.

Reply:
Roger. When https://github.com/nrfconnect/sdk-nrf/pull/9132 is merged, We can remove some of this.